util.js

```diff
@@ -89,18 +89,30 @@ util.sortArrayBasedOnObjectProperty = function (items, prop) {
 */
 util.getVotingResult = function (jiraId, pokerDataModel) {
   console.log('Util getVotingResult : begin')
-  const fibonacci = [1, 2, 3, 5, 8, 13, 21, 34, 55]
+  const fibonacci = [1, 2, 3, 5, 8, 13, 21, 34, '?']
```

Comment:
This constant is also in the `pokerbot.js` file. This should be factored out. I would suggest that this file export the constant and that `pokerbot.js` import it.